docs/Maintain/build-and-start-a-node/07-seed-nodes.md

```diff
@@ -0,0 +1,121 @@
+1. Check the prerequisites [here](./01-prerequisites.md).
+2. Download the node. Here you have two options:
+
+   1. Download the node using the [Docker image](https://hub.docker.com/r/thepowerio/tpnode).
+   2. Download the [source](../../Maintain/build-and-start-a-node/06-startingTpNode_source.md#downloading-and-building-the-node) code and build it (only for advanced users).
+
+3. Install Erlang (this step is applicable for Docker installation **ONLY**):
+
+   Before starting Tea Ceremony you need to set up your environment by installing Erlang. You will need Erlang to run the Tea Ceremony client. You will not be able to start the node and connect to the chain without Tea Ceremony. To install Erlang, run:
+
+   ```bash
+   apt -y install erlang-base erlang-public-key erlang-ssl
+   ```
+
+   > **Note**
+   >
+   > You need to install `erlang-public key` and `erlang-ssl`. Otherwise, Erlang will not operate properly!
+
+4. Get Tea Ceremony client and token
+
+   1. Get the Tea Ceremony client by running the following command:
+
+      ```bash
+      wget https://tea.thepower.io/teaclient
+      ```
+
+   2. Change the `teaclient` file mode to executable by running the following command:
+
+      ```bash
+      chmod +x teaclient
+      ```
+
+      Otherwise, you will NOT be able to start the client.
+
+   3. Get the Tea Ceremony token from the testnet bot.
+
+      :::note
+
+      This and the following steps are crucial because you will NOT be able to start your node without `genesis.txt` and `node.config` files. You can find more information about these files [here](https://doc.thepower.io/docs/Maintain/build-and-start-a-node/tpNodeConfiguration).
+
+      :::
+
+5. Generate private key with **wrong** token:
+
+   1. Start the Tea Ceremony client with wrong token:
+
+      ```bash
+      ./teaclient 123456
+      ```
+
+   2. The Client will respond:
+   
+      ```text
+      Server rejects connection, reason: bad_token
+      ```
+
+   3. You will receive `node.config` file with the private key. Delete all the contents of the file except the key (`privkey` line) (if any content present) and copy the following:
+
+      ```erlang
+      {tpic,#{peers => [],port => 43314}}.
+      {discovery,#{addresses => []}}.
+
+      {privkey,"302E020100300506032B6570042204200011223344556677001122334455667700112233445566770011223344556677"}.
+
+      {replica, true}.
+      {upstream, [
+         <PEER_ADDRESS>
+      ]}.
+
+      {hostname, <Hostname>}.
+
+      {dbsuffix,""}.
+      {loglevel, info}.
+      {info_log, "log/info.log"}.
+      {error_log, "log/error.log"}.
+      {debug_log, "log/debug.log"}.
+      {rpcsport, 1443}.
+      {rpcport, 1080}.
+      ```
+      
+      :::warning
+
+      Use [this]() guide for more information on `node.config`.
+      :::
+
+      :::attention
+
+      You need to replace:
+
+      - `privkey` with your private key; 
+      - `hostname` with your hostname;
+      - `upstream`. It is obtained from address.
```

Comment:
I told obtained from us. We have no idea yet how we will distribute upstreams. So write anything ambiguous here =)